apps/language_server/lib/language_server/providers/completion.ex

```diff
@@ -139,8 +144,25 @@ defmodule ElixirLS.LanguageServer.Providers.Completion do
       module: env.module
     }
 
+    position_to_insert_alias =
+      ElixirSense.Core.Metadata.get_position_to_insert_alias(metadata, line)
+
+    context =
+      case position_to_insert_alias do
+        {line_to_insert_alias, column_to_insert_alias} ->
+          # Don't know why - 1 is needed here
```

Comment:
Please use `SourceFile` function for converting positions from utf8 1 besed to utf16 0 based. It's not trivial and it's already solved